src/apps/vpn/translations/strings.yaml

```
@@ -543,6 +543,24 @@ onboarding:
   panelThreeSubtitle: "Protecting yourself is simple: tap to connect to the VPN and you’re ready to browse safely. Mozilla VPN will start automatically for seamless protection."
   panelFourTitle: Add even more privacy
   panelFourSubtitle: You can set Mozilla VPN to protect you from ads, trackers, and malware. Plus, customize your protection for specific apps.
+  progressBarDataUse:
+    value: Data
+    comment: Label for onboarding progress bar step
+  progressBarMorePrivacy:
+    value: Privacy
+    comment: Label for onboarding progress bar step
+  progressBarAddDevices:
+    value: Devices
+    comment: Label for onboarding progress bar step
+  progressBarGetStarted:
+    value: Start
+    comment: Label for onboarding progress bar step
```

```suggestion
    comment: Label for onboarding progress bar step. It's the final step in the progress, and indicates that the user can now start using the app.
```

This is kind of hard to understand without the video. Feel free to tweak my suggestion.